.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,30 @@
+name: Pull Request
+
+on:
+  pull_request:
+
+  workflow_dispatch:
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Would this only cancel previous runs of the same HEAD commit or all previous runs on the PR? I think we want the latter behavior. I think `github.ref` may work for that and there's also `github.event.pull_request.number`

Reply:
The `group` here is what determines which workflows should be cancelled once a new one is triggered. Using `github.ref` (the branch name) will cancel all previous runs for the given PR, and combining with `github.workflow` (the name of this workflow) will ensure it only cancels _this_ workflow and not others triggered via the same PR.

Reply:
Nice 👍 was just not sure that different commits on the same branch have the same `github.ref`